src/kytos/interfaceInfo.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <k-accordion>
-
+      <k-button :on_click="bt_state_toggle" :title="next_state"></k-button>
```

Comment:
@Alopalao the resulting CSS messed up the button, could you fix this? Example:

![2022-04-08-152137_503x381_scrot](https://user-images.githubusercontent.com/1010796/162499589-4e14bebb-cc2f-4aab-b3bc-daccd2111b23.png)

Reply:
Sorry, I forgot to mention that this branch is derived from the recently pushed PR #23 so it needs the topology PR [#85](https://github.com/kytos-ng/topology/pull/85). Do I create another topology PR?

Reply:
Got it. I see. PR #85 on topology has landed.